www/src/content/blog/2024-12-17-honc-uptime-monitor.mdx

```diff
@@ -16,20 +16,21 @@ In this tutorial, we'll build a serverless website uptime monitoring application
 Overview of the Application
 Our uptime monitor will:
 
-* Monitor multiple websites simultaneously
-* Allow configurable check intervals per website
-* Track response times and HTTP status codes
-* Calculate uptime percentages
-* Provide a RESTful API for managing monitored sites
-* Include a simple web interface to view results
+- Monitor multiple websites simultaneously
```

Comment:
i ran `format` on the astro codebase and that's what changed some of the stuff in this post, just fyi